src/arch/arm/subarch/armv7a/Mybuild

```diff
@@ -8,6 +8,7 @@ module bundle {
 	depends embox.arch.arm.armlib.context
 	depends embox.arch.arm.armlib.interrupt
 	depends embox.arch.arm.armlib.locore
+	depends embox.arch.generic.arch
```

Comment:
You should not a common dependence directly. it is better to add it into mods.conf